std/json.d

```diff
@@ -359,6 +359,80 @@ struct JSONValue
         return type == JSONType.null_;
     }
 
+    /***
+     * Generic type value getter
+     * A convenience getter that will try to convert the underlaying data to the given type.
```

Comment:
The name `get` doesn't suggest any conversion taking place. I'd suggest renaming it.

Reply:
Would `getAs!T` be more intuitive?

Reply:
With a name like `getAs` and verbiage like `* A convenience getter that will try to convert the underlaying data to the given type`, I would expect it to be trying to do things like converting strings to numbers or numbers to booleans. It's a bit misleading if the function is merely a unified way to read a number but does not allow any kind of conversion between different JSON types.

Reply:
We'd need a good rationale for why *conversion* between types should be in `std.json` while we have a dedicated facility for this purpose (`std.conv`).

I suggest keeping the name `get` and changing the description to e.g. "Returns the `JSONValue` contents as the specified D type `T`".

Reply:
We'd need a good rationale for why *conversion* between types should be in `std.json` while we have a dedicated facility for this purpose (`std.conv`).

I suggest keeping the name `get` and changing the description to e.g. "Returns the `JSONValue` contents as the specified D type `T`".

Reply:
After thinking about it some more, `get` should be ok.

Reply:
I will change the comment as CyberShadow suggested nontheless